[2022-05-31 19:15:41,634] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: fhvhv_ingestion_gcs_dag.download_dataset_task scheduled__2020-05-01T06:00:00+00:00 [queued]>
[2022-05-31 19:15:41,656] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: fhvhv_ingestion_gcs_dag.download_dataset_task scheduled__2020-05-01T06:00:00+00:00 [queued]>
[2022-05-31 19:15:41,662] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 19:15:41,669] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-31 19:15:41,678] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 19:15:41,713] {taskinstance.py:1378} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-05-01 06:00:00+00:00
[2022-05-31 19:15:41,736] {standard_task_runner.py:52} INFO - Started process 709 to run task
[2022-05-31 19:15:41,753] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'fhvhv_ingestion_gcs_dag', 'download_dataset_task', 'scheduled__2020-05-01T06:00:00+00:00', '--job-id', '469', '--raw', '--subdir', 'DAGS_FOLDER/fhv_ingestion.py', '--cfg-path', '/tmp/tmpoijy3t0i', '--error-file', '/tmp/tmpgyefiv80']
[2022-05-31 19:15:41,758] {standard_task_runner.py:80} INFO - Job 469: Subtask download_dataset_task
[2022-05-31 19:15:41,925] {task_command.py:370} INFO - Running <TaskInstance: fhvhv_ingestion_gcs_dag.download_dataset_task scheduled__2020-05-01T06:00:00+00:00 [running]> on host 3f188d455d5a
[2022-05-31 19:15:42,015] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-05-31 19:15:42,109] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fhvhv_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-05-01T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-05-01T06:00:00+00:00
[2022-05-31 19:15:42,115] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-31 19:15:42,118] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL https://s3.amazonaws.com/nyc-tlc/trip+data/fhvhv_tripdata_2020-05.parquet > /opt/***/output_fhvhv_2020-05.parquet']
[2022-05-31 19:15:42,154] {subprocess.py:85} INFO - Output:
[2022-05-31 19:16:02,537] {local_task_job.py:221} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2022-05-31 19:16:02,612] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 709. PIDs of all processes in the group: [711, 712, 709]
[2022-05-31 19:16:02,631] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 709
[2022-05-31 19:16:02,640] {taskinstance.py:1542} ERROR - Received SIGTERM. Terminating subprocesses.
[2022-05-31 19:16:02,669] {subprocess.py:103} INFO - Sending SIGTERM signal to process group
[2022-05-31 19:16:02,996] {process_utils.py:75} INFO - Process psutil.Process(pid=712, status='terminated', started='19:15:41') (712) terminated with exit code None
[2022-05-31 19:16:03,141] {process_utils.py:75} INFO - Process psutil.Process(pid=709, status='terminated', exitcode=0, started='19:15:41') (709) terminated with exit code 0
[2022-05-31 19:16:03,144] {process_utils.py:75} INFO - Process psutil.Process(pid=711, status='terminated', started='19:15:41') (711) terminated with exit code None
